/**
 * Created by qfeng on 2018/12/29
 */

syntax = "proto3";

option cc_enable_arenas = true;
option java_outer_classname = "ContFeatGrpPb";
option java_multiple_files = true;
option java_package = "algocomp";

import "featpb/cont_feat_data.proto";

package algocomp;

message ContFeatGrp {
    map<string, ContFeatData> feats = 1;
}